gems/pending/appliance_console/database_replication.rb

```diff
@@ -9,9 +9,8 @@ class DatabaseReplication
 
     REPMGR_CONFIG = '/etc/repmgr.conf'.freeze
 
-    attr_accessor :cluster_name, :node_number,
-                  :database_name, :database_user, :database_password,
-                  :primary_host, :standby_host
+    attr_accessor :cluster_name, :node_number, :database_name, :database_user,
+                  :database_password, :primary_host
```

Comment:
@carbonin Total nit. Not sure how this small change makes the code easier to read. I had lined up all
the `:database_<x>` and `:<x>_host` symbols. I thought it was easier to read that way.